lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/benchmark/python/scipy/benchmark.py

```diff
@@ -105,8 +105,8 @@ def main():
     print_version()
 
     name = ""
-    setup = "from scipy import TODO; from random import random;"
-    stmt = "y = TODO(1, random())"
+    setup = "from scipy.special import eval_hermitenorm; from random import random;"
+    stmt = "y = eval_hermitenorm(2, random()*10 - 5)"
```

Comment:
`random()*10.0 - 5.0`. In Python, decimals matter. Here, the execution context performs casts due to `random()`, but we would prefer to be explicit, similar to how we are in JavaScript.

Reply:
Thanks!  I thought testing a few values for ```n``` was sufficient but I was definitely prepared to do more if you thought it was necessary.